provers/evm-prover/src/main.rs

```diff
@@ -0,0 +1,93 @@
+use sp1_sdk::HashableKey;
+use std::env;
+use std::fs;
+use tonic::{transport::Server, Request, Response, Status};
+
+// Import the generated proto rust code
+pub mod prover {
+    tonic::include_proto!("celestia.prover.v1");
+    tonic::include_proto!("celestia.ibc.lightclients.groth16.v1");
+}
+
+use prover::prover_server::{Prover, ProverServer};
+use prover::{
+    InfoRequest, InfoResponse, ProveStateMembershipRequest, ProveStateMembershipResponse,
+    ProveStateTransitionRequest, ProveStateTransitionResponse,
+};
+use sp1_sdk::{include_elf, ProverClient, SP1ProvingKey};
+
+// The ELF file for the Succinct RISC-V zkVM.
+const BLEVM_ELF: &[u8] = include_elf!("blevm-mock");
```

Comment:
[no change needed] I'm surprised this works b/c `blevm-mock` is defined in a different directory

Reply:
it works because i compiled it in the build.rs